internal/sdkv2provider/resource_user_identity.go

```diff
@@ -45,14 +46,29 @@ func resourceUserIdentity() *schema.Resource {
 					},
 				}
 			},
+			IdentityUpgraders: []schema.IdentityUpgrader{
+				{
+					Version: 1,
+					Type: tftypes.Object{
+						AttributeTypes: map[string]tftypes.Type{
+							"email": tftypes.String,
+						},
+					},
+				},
+			},
+		},
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughWithIdentity("email"),
```

Comment:
Looks like we just need to bump to using the latest commit on main for now 👍🏻

Reply:
oh yeah, my `go.work` shadowed that 😅